src/frontend/core/scheduler/JobScheduler.cpp

```diff
@@ -126,7 +128,9 @@ JobResponse JobScheduler::getResult(JobRequest jobRequest) {
             responseFound = true;
         }
         responseVectorMutex.unlock();
-        usleep(50000);
+        if (!responseFound) {
+            usleep(50000);
```

Comment:
Why is this specifically 50000? why not 70000?

Reply:
I didn't change the value here and it's from there previously.
Here, the sleep is moved into a condition to call the sleep function only if necessary

Reply:
Ok, I see